src/main/java/mate/academy/springbootwebgreqit/dto/CreateBookRequestDto.java

```diff
@@ -1,12 +1,15 @@
 package mate.academy.springbootwebgreqit.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Positive;
 import lombok.Data;
 import java.math.BigDecimal;
 import java.util.List;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank(message = "Title cannot be empty")
```

Comment:
Add `@NotBlank` for all fields, except description and coverImage, add `@NotEmpty` for categoriesIds and `@NotNull` for price